### PR TITLE
--light and --dark options

### LINKED ISSUE #6: Allow restricting to light/dark background themes
cc @clnoll

### PR DESCRIPTION
**Pull request by dandavison**
@clnoll fixes https://github.com/dandavison/iterminator/issues/6

### CODE REVIEW

File: iterminator/iterminator.py [hunk 1]
```diff
@@ -12,6 +12,24 @@
 import getchs
 
 
+LIGHT_SCHEMES = [
```
Comment (clnoll) [inline iterminator/iterminator.py]: how do you know which are light?  is there a way to abstract this, for when new schemes are added?

Comment (dandavison) [inline iterminator/iterminator.py]: That would definitely be cool. Presumably that would involve inspecting the [`*.itermcolors`](https://github.com/mbadolato/iTerm2-Color-Schemes/tree/a646a1de3d2085960d810e77b9011c96ff5884bc/schemes) color scheme file. E.g. `Piatto Light.itermcolors` has this entry, so it does look like this is very doable.

```
        <key>Background Color</key>
        <dict>
                <key>Blue Component</key>
                <real>1</real>
                <key>Green Component</key>
                <real>1</real>
                <key>Red Component</key>
                <real>1</real>
        </dict>
```

Let's create a new issue for doing it properly and allow this one to merge? For this PR I simply hit right arrow lots of times and stopped every time it had a light background.

File: iterminator/iterminator.py [hunk 2]
```diff
@@ -47,14 +65,31 @@ def __init__(self, quiet=True):
...
         self.animation_control = Thread(target=self.control)
         self.paused = False
         self.quitting = False
 
+    def _post_change_schemes_hook(self):
+        self.name_to_scheme = {s.name: s for s in self.schemes}
+        self.scheme_names = [s.name for s in self.schemes]
+        self.blank = ' ' * max(len(s.name) for s in self.schemes)
+
+    def filter(self, scheme_names):
```
Comment (clnoll) [inline iterminator/iterminator.py:79]: cool

**Comment by clnoll** [comment]
Looks great!

File: iterminator/iterminator.py (hunk 1 above)
Comment (clnoll) [inline iterminator/iterminator.py]: Makes sense!

**Comment by dandavison** [comment]
Thanks @clnoll!

**Pull request merged**